tools/mo/openvino/tools/mo/front/onnx/unsqueeze_ext.py

```diff
@@ -18,5 +18,9 @@ class UnsqueezeFrontExtractor(FrontExtractorOp):
     def extract(cls, node):
         axis = int64_array(onnx_attr(node, 'axes', 'ints', default=[]))
 
-        ExpandDims.update_node_stat(node, {'expand_axis': axis})
+        attrs = {
+            'expand_axis': axis if len(axis) != 0 else None
+        }
+
+        ExpandDims.update_node_stat(node, attrs)
```

Comment:
could you please tell - what is wrong with the current implementation?

Reply:
@rkazants I just wanted to match it with squeeze implementation so instead of `'expand_axis': array([], dtype=int64)` we have `'expand_axis=None'`. Do you want me to remove it?

Reply:
@bszmelcz, sorry for confusing in the previous comments if any. Please do not change legacy ONNX FE if no actual reason. Let us revert this change if no issue.

In the previous comment I asked about tests for new ONNX FE. It exists, it is fine to me. But please ask someone from ONNX FE team to review this part as well.